src/Workspaces/Core/Portable/Formatting/ISyntaxFormattingService.cs

```diff
@@ -2,17 +2,22 @@
 
 using System.Collections.Generic;
 using System.Threading;
-using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.Formatting.Rules;
+using Microsoft.CodeAnalysis.Text;
+
+#if !CODE_STYLE
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Options;
-using Microsoft.CodeAnalysis.Text;
+#endif
 
 namespace Microsoft.CodeAnalysis.Formatting
 {
-    internal interface ISyntaxFormattingService : ILanguageService
+    internal interface ISyntaxFormattingService
+#if !CODE_STYLE
+        : ILanguageService
```

Comment:
could we just potentially define this as an internal interface as well in the codestyle layer to simplify this locations?  As an empty marker interface it doesn't seem terribly unclean to do that.

Reply:
➡️ Ended up not doing this to avoid further confusion regarding the Public API files in the Code Style layer.

Reply:
Works for me.